src/OrchardCore.Modules/OrchardCore.Google/Configuration/GoogleOptionsConfiguration.cs

```diff
@@ -75,6 +76,17 @@ public void Configure(string name, GoogleOptions options)
             {
                 options.CallbackPath = settings.CallbackPath;
             }
+
+            // should be removed after February 2.2 patches
```

Comment:
Please file an issue so we don't forget.

Reply:
filed https://github.com/OrchardCMS/OrchardCore/issues/3000